js2py/internals/opcodes.py

```diff
@@ -798,7 +798,7 @@ def eval(self, ctx):
 g = ''
 for g in globals():
     try:
-        if not issubclass(globals()[g], OP_CODE) or g is 'OP_CODE':
+        if not issubclass(globals()[g], OP_CODE) or g is OP_CODE:
             continue
```

Comment:
`g` is a string, so we need either 
1. `g == "OP_CODE"`   
or
2. `globals()[g] is OP_CODE`